openapi.yml

```diff
@@ -3384,6 +3386,76 @@ paths:
           in: query
           name: created_after
           description: Restricts results to only those that were created after the provided date
+  '/locations/{location_id}/max_capacities':
+    parameters:
+      - schema:
+          type: string
+        name: location_id
+        in: path
+        required: true
+    get:
+      summary: Get the maximum capacity details for a location
+      tags:
+        - MaxCapacities
+      responses:
+        '200':
+          description: 'Successful response returns the current number of signins, expected invites, max_capacity of the location and expected invites for the next 6 hours.'
+          content:
+            application/json:
+              schema:
+                $ref: ./models/MaxCapacityResponse.v1.yaml
+              examples:
+                default:
+                  value:
+                    signins_count: 10
+                    invites_expected_next_hour: 7
+                    max_capacity: 50
+                    chart_data:
+                      - 10
+                      - 5
+                      - 3
+                      - 6
+                      - 5
+        '400':
+          description: A generic error
+          content:
+            application/json:
+              schema:
+                $ref: ./models/ErrorsList.v1.yaml
+              examples:
+                default:
+                  value:
+                    errors:
+                      - domain: some text
+                        attribute: some text
+                        code: some text
+                        message: some text
+                      - domain: some text
+                        attribute: some text
+                        code: some text
+                        message: some text
+        '401':
+          description: "You don't have permission to view the details of the location's capacity."
+        '403':
+          description: You do not have permission for this action
+      operationId: getLocationMaximumCapacity
+      description: Gets the details of the capacity in a location.
+      parameters:
+        - schema:
+            type: string
+          in: query
+          name: local_time
+          description: Current local time with offset value.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: ./models/MaxCapacity.v1.yaml
```

Comment:
So the objective of this endpoint is to set what the local time is in a city? How does that work?

Reply:
We retrieve the local time with the offset value as the input and we convert it to the UTC from the backend application.

Reply:
In that case, the API should be letting the user set the timezone of the location on the location configuration. UTC offsets change for a geolocation on a regular basis, so we can't trust that.

The timezone should be in IANA tz database format, eg: `America/Vancouver`, `Europe/London`, `Asia/Dubai`, etc.

Reply:
Ok. So in the Postgres query execution, if the DateTime is passed with the offset value, it takes care of all the timezone issues. So I am not converting it to UTC now.

So just to clarify, you expect the user to set the timezone whenever they load the capacity screen on their phone?

Reply:
Not every time, but we need to know the IANA TZ entry for a location before we can accurately know the time there. Knowing that accurate time is a prerequisite for capacity management. Once we have that recorded for the location, we won't need to ask any timezone related questions of the user.

Reply:
ok 👍